frontend/src/metabase/selectors/selectors.unit.spec.ts

```diff
@@ -12,38 +12,42 @@ import { getIsPaidPlan, getUpgradeUrl } from "./settings";
 describe("getUpgradeUrl", () => {
   it.each([
     {
-      media: "license",
+      content: "license",
       users: 1,
       features: { hosting: false, sso_jwt: false },
-      params: "?utm_media=license&utm_source=oss&utm_users=1",
+      params:
+        "?utm_source=product&utm_medium=upsell&utm_content=license&source_plan=oss&utm_users=1",
```

Comment:
Upgrade URL change. Please see the issue [description how it has changed](https://github.com/metabase/metabase/pull/47116#issue-2480439695).